editor/export/editor_export_platform.cpp

```diff
@@ -223,6 +223,10 @@ Error EditorExportPlatform::_save_pack_file(void *p_userdata, const String &p_pa
 	PackData *pd = (PackData *)p_userdata;
 
 	String simplified_path = p_path.simplify_path();
+	if (simplified_path.begins_with("uid://")) {
+		simplified_path = ResourceUID::ensure_path(simplified_path).simplify_path();
```

Comment:
```suggestion
		simplified_path = ResourceUID::uid_to_path(simplified_path).simplify_path();
```
`ensure_path()` should only be used when the path can be either uid or not. You are already checking this.